l10n/sleha/ja-jp/xml/ha_maintenance.xml

```diff
@@ -67,7 +75,7 @@
    </listitem>
    <listitem>
     <para>
-     リソースの停止が失敗したりタイムアウトしたりする場合、STONITHメカニズムはそのノードをフェンシングし、これをシャットダウンします。
+     リソースの停止が失敗したりタイムアウトしたりする場合、&stonith;メカニズムはそのノードをフェンシングし、これをシャットダウンします。
```

Comment:
Oh, here we have entities all of a sudden? (while usually with `daps locdrop` the entities are resolved). This might trigger some problems when generating the output formats (in case the entities in this file are resolved with their English equivalents now). 

Will run some checks and discuss with @jfaltenbacher offline first of all.

Reply:
the entities are not resolved because only the file was handed over to translation without loc drop. Let's discuss offline